### PR TITLE
More info

### PR DESCRIPTION
**Pull request by balloob**
Add more info demo

### CODE REVIEW

File: gallery/src/components/demo-more-info.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import '../../../src/components/ha-card.js';
+
+
+class DemoMoreInfo extends PolymerElement {
+  static get template() {
+    return html`
+        <style>
+          :host {
+            display: flex;
+            align-items: start;
```
Comment (c727) [inline gallery/src/components/demo-more-info.js]: default value

File: gallery/src/components/demo-more-info.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            in-dialog
+          ></state-card-content>
+
+          <more-info-content
+            hass='[[hass]]'
+            state-obj='[[_stateObj]]'
+          ></more-info-content>
+        </ha-card>
+        <template is='dom-if' if='[[showConfig]]'>
+            <pre>[[_jsonEntity(_stateObj)]]</pre>
```
Comment (c727) [inline gallery/src/components/demo-more-info.js]: indentation

File: gallery/src/components/demo-more-infos.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+class DemoMoreInfos extends PolymerElement {
+  static get template() {
+    return html`
+      <style>
+        .cards {
+          display: flex;
+          flex-wrap: wrap;
+          justify-content: center;
+          align-items: start;
```
Comment (c727) [inline gallery/src/components/demo-more-infos.js]: default value

File: gallery/src/ha-gallery.js [hunk 4]
```diff
@@ -31,10 +31,21 @@ class HaGallery extends PolymerElement {
         visibility: hidden;
       }
 
-      paper-card {
+      .pickers {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: center;
+        align-items: start;
```
Comment (c727) [inline gallery/src/ha-gallery.js]: default

File: gallery/src/ha-gallery.js [hunk 5]
```diff
@@ -151,6 +188,14 @@ class HaGallery extends PolymerElement {
   _backTapped() {
     document.location.hash = '';
   }
+
+  _computeLovelace(dems) {
+    return dems.filter(demo => demo.includes('hui'));
+  }
+
+  _computeMoreInfos(dems) {
```
Comment (c727) [inline gallery/src/ha-gallery.js]: demos

File: gallery/src/ha-gallery.js [hunk 6]
```diff
@@ -151,6 +188,14 @@ class HaGallery extends PolymerElement {
   _backTapped() {
     document.location.hash = '';
   }
+
+  _computeLovelace(dems) {
```
Comment (c727) [inline gallery/src/ha-gallery.js]: demos

File: gallery/src/components/demo-more-info.js (hunk 1 above)
Comment (balloob) [reply gallery/src/components/demo-more-info.js]: Without I get this:

image

Comment (balloob) [reply gallery/src/components/demo-more-info.js]: (the more info card is stretched to the height of the `<pre>`)

File: gallery/src/ha-gallery.js (hunk 4 above)
Comment (balloob) [reply gallery/src/ha-gallery.js]: image

Comment (balloob) [reply gallery/src/ha-gallery.js]: (the default is 'normal')

File: gallery/src/ha-gallery.js (hunk 6 above)
Comment (balloob) [reply gallery/src/ha-gallery.js]: That was to avoid shadowing the `demos` var at the root level of the module. I have renamed that one now to `DEMOS`

**Comment by c727** [comment]
Ok sorry, only did a small test in browser.

Ship it?! :+1:

**Pull request merged**